test/scenarios-bootc/releases/el96-y1@el96-lrel@lvms.sh

```diff
@@ -44,15 +44,8 @@ scenario_run_tests() {
         return 0
     fi
 
-    # Verify if microshift has started up fully
-    local vmname="host1"
-    local -r full_vmname="$(full_vm_name "${vmname}")"
-    local -r vm_ip="$(get_vm_property "${vmname}" ip)"
-    if ! wait_for_greenboot "${full_vmname}" "${vm_ip}"; then
-        record_junit "${vmname}" "pre_test_greenboot_check" "FAILED"
-        return 1
-    fi
-    record_junit "${vmname}" "pre_test_greenboot_check" "OK"
+    # Setup oc client and kubeconfig for gingko tests
+    setup_oc_and_kubeconfig host1
```

Comment:
Losing the explicit wait_for_greenboot looks non-obvious to me. Also this introduces an implicit waiting in places where it was not present.